.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: 🐞 Bug report
+about: Create a report to help us improve Cryostat
+title: '[Bug] <title>'
+labels: bug, needs triage
```

Comment:
Might need to create `needs triage` label? Or we can just remove it.

Reply:
I think we would have to add it, I believe it makes sense to keep it if we start to have more contributors from outside.

Reply:
I think it's a fine label to add.

We could probably also update the `pr-ci.yml` to both require the `safe-to-test` label to be present *and* and `needs triage` label to *not* be present before it runs any tests, just as an additional safeguard.

Reply:
I was thinking that the `needs triage` label would only be for issues, as a way for a cryostat reviewer to figure out if the issue needs to be assign a priority to the issue.

Reply:
It might be nice to have for PRs too as a way to highlight PRs from external contributors that need someone to engage with the author, review the PR, and decide if it's safe to remove the `needs triage` and add `safe-to-test`.